ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js

```diff
@@ -31,31 +32,30 @@ describe('Network Graph Search', () => {
     withAuth();
 
     it('should filter to show only the deployments from the stackrox namespace and deployments connected to them', () => {
```

Comment:
I wonder if there is value in having this test
`'should filter to show only the deployments from the stackrox namespace and deployments connected to them'`
and the following test
`'should filter to show only the stackrox namespace and deployments connected to stackrox namespace'`
at all.

With the current default behavior of a fresh install on a cluster locally and in CI, the graph will never show deployments or namespaces other than "stackrox" unless "Show Orchestrator Components" is enabled. The assertions are then checking to be sure the data falls into the only possible category it could be in.

It might be worth changing these in the future to use mocked data that includes multiple namespaces with deployments so we can test that the data is truly filtered by namespace.

Reply:
Yes, good point.